packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testIsCloser.st

```diff
@@ -0,0 +1,17 @@
+tests - state
+testIsCloser
```

Comment:
Maybe 'testClosingDelimiterIsCloser' and 'testNotClosingDelimiterIsNotCloser', only using the method name as the name for the test case can be quite confusing (as it is in this case, cause like, is the test coming closer or is it further away? xD)